doc/tutorials/dnn/dnn_android/dnn_android.markdown

```diff
@@ -93,14 +38,39 @@ range.
 frames. Modify `app/src/main/res/layout/activity_main.xml`:
 @include android/mobilenet-objdetect/res/layout/activity_main.xml
 
-- Put downloaded `MobileNetSSD_deploy.prototxt` and `MobileNetSSD_deploy.caffemodel`
-into `app/build/intermediates/assets/debug` folder.
+- Put downloaded `deploy.prototxt` and `mobilenet_iter_73000.caffemodel`
+into `app/src/main/res/raw` folder.
 
 - Modify `/app/src/main/AndroidManifest.xml` to enable full-screen mode, set up
 a correct screen orientation and allow to use a camera.
-@include android/mobilenet-objdetect/gradle/AndroidManifest.xml
-
-- Replace content of `app/src/main/java/org/opencv/samples/opencv_mobilenet/MainActivity.java`:
+```
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application
+        android:label="@string/app_name">
+        <activity
+            android:exported="true"
+            android:name=".MainActivity"
+            android:screenOrientation="landscape">  <!--Screen orientation-->
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+    </application>
+    <!--Allow to use a camera-->
+    <uses-permission android:name="android.permission.CAMERA"/>
+    <uses-feature android:name="android.hardware.camera" android:required="false"/>
+    <uses-feature android:name="android.hardware.camera.autofocus" android:required="false"/>
+    <uses-feature android:name="android.hardware.camera.front" android:required="false"/>
+    <uses-feature android:name="android.hardware.camera.front.autofocus" android:required="false"/>
+
+</manifest>
+```
+
+
+- Replace content of `app/src/main/java/com/example/myapplication/MainActivity.java` and set package as `package com.example.myapplication;`:
```

Comment:
Please add details on model loading and in-memory parsing. It's not popular for desktop apps.